### ISSUE TITLE
Using C64-Debugger on MacOS

### ISSUE THREAD

**Issue by MihaMarkic** [rolandshacks/vs64#15]
Hi guys, I'm on MacOS Catalina 10.15.3 and I wonder whether anybody can use the C64 Debugger at all.
Download it I get only _C64 Debugger.app_ file (which doesn't run for some reason anyway, while C65XE Debugger.app at least runs)

**Comment by Dids** [comment]
Try an older build from [here](https://csdb.dk/release/?id=170893) for example, then set the debugger path in your `settings.json` appropriately, like so:

```json
"c64.debuggerPath": "/Applications/C64Debugger.app/Contents/MacOS/C64Debugger",
```

This way it won't at least crash, unlike the latest versions on SourceForge.

**Comment by MihaMarkic** [comment]
Thanks for the suggestion.

**Comment by kfox** [comment]
@MihaMarkic I managed to get this extension to work with the C64 Debugger on macOS Catalina 10.15 using the latest build (v0.64.58.4 when I tried it) of [the C64 Debugger from SourceForge](https://sourceforge.net/projects/c64-debugger/files/). Here are the steps I followed:
1. Download and extract the files
2. Copy just the extracted `C64 Debugger.app` directory to another directory such as `/usr/local/bin`
3. In the vs64 extension settings, enable the debugger and set the debugger path to `/usr/local/bin/C64 Debugger.app/Contents/MacOS/C64 Debugger`
4. Open an Assembly source file, hit Ctrl-Alt-P, and select *C64: Debug* and your code should load in the debugger

Hope this helps! 😄
